contracts/CrossChain.sol

```diff
@@ -48,6 +77,16 @@ contract CrossChain is System, ICrossChain, IParamSubscriber{
   event enableOrDisableChannel(uint8 indexed channelId, bool isEnable);
   event addChannel(uint8 indexed channelId, address indexed contractAddr);
 
+  // BEP-171: Security Enhancement for Cross-Chain Module
+  event SubmitEmergencyProposal(bytes32 indexed prosalNameHash, address indexed proposer, uint128 approveThreshold, uint128 expiredAt);
+  event EmergencySuspend(address indexed excutor);
+  event Reopen(address indexed excutor);
+  event Challenge(
```

Comment:
There is a function called `challenge` and an event called  `Challenge`,  this may cause some ABI tools to failed. `SuccessChallenge` is a better option.

Reply:
renamed